src/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Document</title>
+  </head>
+
+
+  <body>
+
+      Hello, world!
+
+  </body>
+
+
+</html>
```

Comment:
слишком много отступов, столько не нужно
```suggestion
  </head>
  <body>
      Hello, world!
  </body>
</html>
```